tools/ansible/tasks/wsu/test/e2e/wsu_test.go

```diff
@@ -254,3 +260,13 @@ func testWorkerLabelsArePresent(t *testing.T) {
 	assert.Contains(t, windowsNodes.Items[0].Labels, workerLabel,
 		"expected worker label to be present on the Windows node but did not find any")
 }
+
+// testHybridOverlayAnnotations tests that the correct annotations have been added to the bootstrapped node
+func testHybridOverlayAnnotations(t *testing.T) {
+	windowsNodes, err := k8sclientset.CoreV1().Nodes().List(metav1.ListOptions{LabelSelector: windowsLabel})
+	require.NoError(t, err, "Could not get list of Windows nodes")
+	assert.Equalf(t, len(windowsNodes.Items), 1, "expected one windows node to be present but found %v",
+		len(windowsNodes.Items))
+	assert.Contains(t, windowsNodes.Items[0].Annotations, hybridOverlaySubnet)
+	assert.Contains(t, windowsNodes.Items[0].Annotations, hybridOverlayMac)
+}
```

Comment:
Ideally we should also be testing if the `BaseOpenShiftNetwork` and `OpenShiftNetwork` HNS networks were created on the node. Not sure if it is easy to do that without running `Get-HnsNetwork` remotely.